contracts/cw-proposal-single/src/msg.rs

```diff
@@ -32,6 +32,9 @@ pub struct InstantiateMsg {
     /// proposal. None if there is no deposit requirement, Some
     /// otherwise.
     pub deposit_info: Option<DepositInfo>,
+    /// Failed proposal executions will be marked failed,
+    /// instead of reverted state, if this feature is enabled
```

Comment:
Same comments for here, `ExecuteMsg`, and `Config` on the docstring if you'd like to take the suggested one.